datahub-web-react/src/app/entity/shared/components/styled/search/EmbeddedListSearchResults.tsx

```diff
@@ -10,14 +10,9 @@ import { EntityAndType } from '../../../types';
 
 const SearchBody = styled.div`
     height: 100%;
-    overflow-y: scroll;
```

Comment:
actually this will break styling for the Add To Assets modal (where we list entities to select from)